apps/docs-snippets/src/guide/contracts/read-only-calls.test.ts

```diff
@@ -13,8 +13,7 @@ describe(__filename, () => {
   it('should successfully execute a read only call', async () => {
     // #region read-only-call-1
     const { value } = await contract.functions.echo_u8(15).get();
-
-    expect(value).toEqual(15);
     // #endregion read-only-call-1
+    expect(value).toEqual(15);
```

Comment:
Was this on purpose?

Why should it be removed from the docs code snippet?

Reply:
_Personally_ I like to keep assertions out of the snippet to focus on the functionality that the docs are referencing rather than an assertion. Potentially a weak arugment, but if someone is not used to jest syntax then this additional line is confusing.